run/opencl/DES_bs_finalize_keys_kernel.cl

```diff
@@ -20,6 +22,7 @@
 #define kvshl1 vshl1
 #define kvshl vshl
 #define kvshr vshr
+#define vlut3 lut3
```

Comment:
I don't think defining `vlut3` clarifies anything here - we could as well use `lut3` in one place below directly.